submissions/Semka094/hooli-style-popup/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap"
+          rel="stylesheet">
+    <title>Pop-up</title>
+</head>
+
+<body>
+
+<div class="menuContainer">
+    <nav class="navWrapper">
+        <div class="navItem">
+            <a class="navLink" href="#">Mail</a>
+        </div>
+        <div class="navItem">
+            <a class="navLink" href="#">Images</a>
+        </div>
+        <div class="navItem icon">
+            <input class="popupBtnCheckbox" type="checkbox" id="popupBtn">
+            <label class="popupBtnLabel" for="popupBtn">
+                <img class="navLink" src="./images/popup-button.png" alt="dotted button">
+            </label>
+            
+            <div class="appsPopup">
+                <div class="popupContainer">
+                    <div class="popupWrapper">
+                        <div class="itemList">
+                            <a href="#" class="columnItem">
+                                <div class="imgItem"><img src="./images/calendar.png" alt="Calendar"></div>
+                                <div class="itemTitle">Calendar</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/mail.png" alt="Mail"></div>
+                                <div class="itemTitle">Mail</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/news.png" alt="News"></div>
```

Comment:
Please increase the font size for better readability.
![image](https://user-images.githubusercontent.com/16196199/187025408-4032c497-1c59-4465-9959-95472e5d92ed.png)